### PR TITLE
120 introduction and origins

### LINKED ISSUE #120: Introduction and Origins
The 2017 Rec doc has an origins/history section - https://www.w3.org/TR/vocab-ssn/#Developments 
For the update, this will need to be supplemented by some material around OMS and STA. 
@KathiSchleidt @sgrellet @alexrobin can you help?

### PR DESCRIPTION
**Pull request by dr-shorthair**
Combined and streamlined Introduction and Origins sections, as well as change-log
Closes #120

### CODE REVIEW

**Review by ldesousa — APPROVED** [review]
The text can still be (slightly) improved here and there for clarity. But better merge these changes now and conduct a comprehensive review later on.

**Comment by dr-shorthair** [comment]
@KathiSchleidt @sgrellet I went ahead and did a brief version of the history of OMS (you guys did not seem to be getting to it). 
Is this OK for now?

**Comment by dr-shorthair** [comment]
Moved Origins to appendix - preview 
https://raw.githack.com/w3c/sdw-sosa-ssn/120-introduction-and-origins/ssn/index.html#origins

File: ssn/frag/Origins.html [hunk 1]
```diff
@@ -1,119 +1,75 @@
...
+    module.
+    SOSA acts as the central building block for the SSN and is packaged with very lightweight axiomatization so as to
+    support use by web-developers.
+</p>
+
+<p>For details on the design philosophy and decisions around the SSN Ontology, please consult chapter "3. Origins of
+    SSN and SOSA" in [[[vocab-ssn-20171019]]].
+</p>
+
+<h4>New edition</h4>
```
Comment (sgrellet) [inline ssn/frag/Origins.html]: suggestion to move it plural "New edition**s**'  

as we speak about both OMS and SSN Ontology new editions

File: ssn/frag/Introduction.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<p>
+  Sensors are a major source of data available on the Web today.
+  However, searching, reusing, integrating, and interpreting sensor data requires more than just the sensing results.
+  For the proper interpretation of these values, knowledge of the feature of interest, such as a river, the observed
+  property, such as flow velocity, the sampling strategy, such as the specific locations and
+  times at which the velocity was measured, and a variety of other information are required.
+</p>
+<p>
+  OGC's Sensor Web Enablement standards [[SWE]], [[SensorML]], [[OandM]], [[OMS]], [[STA]] provide
+  conceptual models, as well as some encodings for the annotation of sensors and their observations.
```
Comment (sgrellet) [inline ssn/frag/Introduction.html:13]: suggestion to add : "Section 7 of [[OMS]] provides a comprehensive description of the characteristics of observations and samples"
see : https://docs.ogc.org/as/20-082r4/20-082r4.html#_fundamental_characteristics_of_observations_and_samples_informative

**Review by sgrellet — APPROVED** [review]
just 2 minors suggestion.

but overall fine with this version

File: ssn/frag/Origins.html (hunk 1 above)
Comment (dr-shorthair) [reply ssn/frag/Origins.html]: Accepted

File: ssn/frag/Introduction.html (hunk 2 above)
Comment (dr-shorthair) [reply ssn/frag/Introduction.html:13]: I omitted 'Section 7 of' - too much for the introduction. 
But I added the specific pointer to Section 7 in the Origins narrative.

**Pull request merged**